app/models/topic_user.rb

```diff
@@ -296,8 +298,6 @@ def track_visit!(topic_id, user_id)
                                    FROM topic_users AS ftu
                                    WHERE ftu.user_id = :user_id and ftu.topic_id = :topic_id)"
 
-    INSERT_TOPIC_USER_SQL_STAFF = INSERT_TOPIC_USER_SQL.gsub("highest_post_number", "highest_staff_post_number")
```

Comment:
No longer required because the SQL query doesn't include `highest_post_number` anymore.